sambaplayersdk/src/main/java/com/sambatech/player/utils/SharedPrefsUtils.java

```diff
@@ -0,0 +1,44 @@
+package com.sambatech.player.utils;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.preference.PreferenceManager;
+import android.text.TextUtils;
+
+final public class SharedPrefsUtils {
+
+    public class SharedPrefsKeys {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this a "static" inner class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.sambatech.com/coding_rules#rule_key=squid%3AS2694)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.sambatech.com/coding_rules#rule_key=squid%3AS1118)